From 58cb1e280df20f2e2027f31e27a00e2d1fe29bc7 Mon Sep 17 00:00:00 2001
From: James Hunt <ohthatjames@gmail.com>
Date: Thu, 25 Jul 2024 18:21:53 +0100
Subject: [PATCH] Remove duplicate key in auth_settings hash

This was raising a warning. The sections are duplicates so it's safe to
remove one.
---
 lib/slipstream_client/configuration.rb | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/lib/slipstream_client/configuration.rb b/lib/slipstream_client/configuration.rb
index 7abfc28..1806663 100644
--- a/lib/slipstream_client/configuration.rb
+++ b/lib/slipstream_client/configuration.rb
@@ -234,13 +234,6 @@ def basic_auth_token
     # Returns Auth Settings hash for api client.
     def auth_settings
       {
-        'azure_auth' =>
-          {
-            type: 'oauth2',
-            in: 'header',
-            key: 'Authorization',
-            value: "Bearer #{access_token_with_refresh}"
-          },
         'azure_auth' =>
           {
             type: 'oauth2',auth_settings
